### DISCUSSION TITLE
where do i save skills to

### DISCUSSION THREAD

**Discussion by geetee24** [OpenVoiceOS/ovos-buildroot#225] [Q&A] [answered]
new to ovos.   expert in Mycroft.

what is the path to where I can copy my mC skills into ovos?

and is this the manual? https://openvoiceos.github.io/ovos-technical-manual/osm/

**Comment by goldyfruit** [comment]
OSM has been deprecated so we recommend to convert legacy skills to Python package, see: https://github.com/mikejgray/ovos-skill-projen

**Comment by JarbasAl** [comment, marked as answer]
Existing mycroft skills will keep working until version `0.1.0`

this means you can copy them to the **NOT RECOMMENDED** path of `/opt/mycroft/skills` (you might run into permission issues) or to the XDG compliant equivalent in OVOS `~/.local/share/mycroft/skills`

skills here should load just like in classic mycroft, and both `MycroftSkill` and `OVOSSkill` should be supported

release `0.0.8`  (the next one) is the last that will support this, moving to properly installable skills will be required to upgrade to `0.1.0`

https://github.com/mikejgray/ovos-skill-projen should help you modernize any skills you have

developer docs can be found here: https://openvoiceos.github.io/ovos-technical-manual
